### DISCUSSION TITLE
Custom template - hint please :D

### DISCUSSION THREAD

**Discussion by DuckerQA** [rochacbruno/marmite#116] [General]
I'm currently playing around with creating custom template using marmite and I'm wondering how I can add tags from group.html to list.html so that they're available on the top on the blog and not in a separate tab. The biggest problem for me is that group_content is not available for list.html and I don't want to modify site.rs. Any hints ? :D

**Comment by rochacbruno** [comment]
Hi @DuckerQA right now the only way would be by embedding using iframe, which is not a good solution.

We can add grouped_content to the global context, there is not huge impact on doing it, I can add it but will be available only next week.

Another option would be you iterating `site_data.posts` and performing the groupby tag in the template https://keats.github.io/tera/docs/#group-by

**Reply by DuckerQA** [reply]
The second solution was perfect, thanks a lot for your help! If it's not a problem to make grouped_content to global context would be great, I think it will make Marmite more flexible in terms of editing custom templates ! Thanks for help!
